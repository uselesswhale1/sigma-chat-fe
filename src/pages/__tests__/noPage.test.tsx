import { render } from "@testing-library/react";
import { NoPage } from "../noPage";

describe("NoPage page should", () => {
  it("renders", () => {
    const { container } = render(<NoPage />);

    expect(container).toMatchSnapshot();
  });
});
