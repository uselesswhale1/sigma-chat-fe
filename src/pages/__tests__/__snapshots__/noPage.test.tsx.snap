// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPage page should renders 1`] = `
<div>
  Page not found (ha-ha)
</div>
`;
